### PR TITLE
Added the store

### PR DESCRIPTION
**Pull request by athiruma**
redux-store:
- To manage the state components centrally
- Added the state to the LocalMonitoring Component.

### CODE REVIEW

**Comment by bhanvimenghani** [comment]
@athiruma have you tested the pr somewhere (minikube/openshift)? I can see some errors while running on minikube

File: src/app/Analytics/LocalMonitoring/DatasourceTable.tsx [hunk 1]
```diff
@@ -106,7 +95,10 @@ const DatasourceTable = (props: { fetchDatasourcesData }) => {
             ))}
           </Tbody>
         </Table>
-        {isComponentVisible && <ClusterGroupTables clusterGroupData={clusterGroupData} dsname={dsname} />}
+
+        {dataLoaded && dsname.length > 0 &&  <ClusterGroupTables clusterGroupData={dataSourceSelector.dataSourceMetaData}  dsname={dsname}/>}
+      </div>
```
Comment (bhanvimenghani) [inline src/app/Analytics/LocalMonitoring/DatasourceTable.tsx:100]: remove the closing tag as there is no opening div tag.

**Review by bhanvimenghani — CHANGES_REQUESTED** [review]
I cant see the UI holding the state, when i go back to some other tab the state is lost.

File: src/app/Analytics/LocalMonitoring/DatasourceTable.tsx (hunk 1 above)
Comment (athiruma) [reply src/app/Analytics/LocalMonitoring/DatasourceTable.tsx:100]: Ah, thanks! I changed it but forgot to push the changes.

**Comment by bhanvimenghani** [comment]
lgtm!

**Review by bhanvimenghani — APPROVED** [review]

**Comment by dinogun** [comment]
@athiruma Please make sure that all your commits are GPG signed. Also it would be good to not use force push as that maintains change history and is easier to review incremental changes

**Pull request merged**